libs/damap/src/lib/components/dmp/summary/summary.component.html

```diff
@@ -9,7 +9,16 @@
     <th mat-header-cell *matHeaderCellDef>
       {{ "dmp.steps.summary.table.header.step" | translate }}
     </th>
-    <td mat-cell *matCellDef="let element">{{ element.step | translate }}</td>
+    <td mat-cell *matCellDef="let element; let i = index">
+      <a
+        (click)="navigateToStep(i)"
+        matTooltip="Jump to section"
```

Comment:
The tooltip does not show up for me. Does it work for you?

Reply:
Fixed it, somehow the change to the modules file was not commited. I slightly altered the text of the tooltip and the aria-label as well.